src/patternfly/components/Skeleton/skeleton.scss

```diff
@@ -0,0 +1,217 @@
+  --pf-c-skeleton__element--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-skeleton__element--after--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-skeleton__element--after--LinearGradientAngle: 90deg;
+  --pf-c-skeleton__element--after--LinearGradientColorStop1: transparent;
+  --pf-c-skeleton__element--after--LinearGradientColorStop2: #ededed;
+  --pf-c-skeleton__element--after--LinearGradientColorStop3: transparent;
+  --pf-c-skeleton__element--after--TranslateX: -100%;
+  --pf-c-skeleton__element--after--AnimationName: loading;
+  --pf-c-skeleton__element--after--AnimationDuration: 2s;
+  --pf-c-skeleton__element--after--AnimationIterationCount: infinite;
+  --pf-c-skeleton__element--after--AnimationTimingFunction: linear;
+  --pf-c-skeleton__element--after--AnimationDelay: .5s;
+
+  // Circle
+  --pf-c-skeleton__element--m-circle--BorderRadius: var(--pf-global--BorderRadius--lg);
+
+  // Static width and height
+  --pf-c-skeleton__element--m-width-sm--Width: #{pf-size-prem(100px)};
+  --pf-c-skeleton__element--m-width-md--Width: #{pf-size-prem(200px)};
+  --pf-c-skeleton__element--m-width-lg--Width: #{pf-size-prem(300px)};
+  --pf-c-skeleton__element--m-height-sm--Height: #{pf-size-prem(100px)};
+  --pf-c-skeleton__element--m-height-md--Height: #{pf-size-prem(200px)};
+  --pf-c-skeleton__element--m-height-lg--Height: #{pf-size-prem(300px)};
+
+
+  // Text modifiers
+  --pf-c-skeleton__element--m-text-4xl--Height: calc(var(--pf-global--FontSize--4xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-3xl--Height: calc(var(--pf-global--FontSize--3xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-2xl--Height: calc(var(--pf-global--FontSize--2xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-xl--Height: calc(var(--pf-global--FontSize--xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-lg--Height: calc(var(--pf-global--FontSize--lg) * var(--pf-global--LineHeight--md));
+  --pf-c-skeleton__element--m-text-md--Height: calc(var(--pf-global--FontSize--md) * var(--pf-global--LineHeight--md));
+  --pf-c-skeleton__element--m-text-sm--Height: calc(var(--pf-global--FontSize--sm) * var(--pf-global--LineHeight--md));
+
+  // Width
+  --pf-c-skeleton__element--m-width-10--Width: 10%;
+  --pf-c-skeleton__element--m-width-15--Width: 15%;
+  --pf-c-skeleton__element--m-width-20--Width: 20%;
+  --pf-c-skeleton__element--m-width-25--Width: 25%;
+  --pf-c-skeleton__element--m-width-30--Width: 30%;
+  --pf-c-skeleton__element--m-width-35--Width: 35%;
+  --pf-c-skeleton__element--m-width-40--Width: 40%;
+  --pf-c-skeleton__element--m-width-45--Width: 45%;
+  --pf-c-skeleton__element--m-width-50--Width: 50%;
+  --pf-c-skeleton__element--m-width-60--Width: 60%;
+  --pf-c-skeleton__element--m-width-70--Width: 70%;
+  --pf-c-skeleton__element--m-width-80--Width: 80%;
+  --pf-c-skeleton__element--m-width-90--Width: 90%;
+
+  &.pf-m-padding {
```

Comment:
do we need the parent element, and this modifier?

Reply:
True, I was thinking for elements that didnt have padding. But maybe we remove this and if there is a use case where the component needs padding we could add it back in